src/test/ui/symbol-names/x86-stdcall.rs

```diff
@@ -0,0 +1,13 @@
+// check-pass
```

Comment:
```suggestion
// build-pass
```
`check-pass` won't run codegen and linking stages.